src/Sylius/Component/Core/Model/Order.php

```diff
@@ -208,6 +209,7 @@ public function getItemUnitsByVariant(ProductVariantInterface $variant): Collect
      */
     public function getPayments(): Collection
     {
+        /** @phpstan-ignore-next-line */
         return $this->payments;
```

Comment:
As above, why we need to ignore this?

Reply:
```
Method Sylius\Component\Core\Model\Order::getPayments() should return Doctrine\Common\Collections\Collection<(int|string),
         Sylius\Component\Payment\Model\PaymentInterface> but returns Doctrine\Common\Collections\Collection<(int|string), Sylius\Component\Core\Model\PaymentInterface>.
```
Obviously here it is right to return a list of core payment, it sounds like a PHPStan bug...